pom.xml

```diff
@@ -166,9 +104,9 @@
   </properties>
 
   <scm>
-    <connection>scm:git:git@github.com:apache/incubator-hudi.git</connection>
-    <developerConnection>scm:git:git@github.com:apache/incubator-hudi.git</developerConnection>
-    <url>git@github.com:apache/incubator-hudi.git</url>
+    <connection>scm:git:https://gitbox.apache.org/repos/asf/incubator-hudi.git</connection>
```

Comment:
Why this change? Don't you want contributors to work with the github repo?

Reply:
@tweise : To be compliant for the voting process, I looked into how the parent pom (apache-21.pom) is setup. The corresponding codebase for apache-21.pom is in github but they had similar setup : https://github.com/apache/maven-apache-parent/blob/master/pom.xml . 

I was not aware of the implication of this change. I am happy to revert the scm configuration as we want developers to use github.  Will update the PR.

Reply:
@tweise: Reverted this change. Thanks